src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/Views/DynamicTableViewHeadlineWithImageCell.swift

```diff
@@ -43,6 +46,8 @@ class DynamicTableViewHeadlineWithImageCell: UITableViewCell {
 		backgroundImageView.contentMode = .scaleAspectFit
 		contentView.addSubview(backgroundImageView)
 
+		topInsetConstraint = headlineLabel.topAnchor.constraint(equalTo: contentView.topAnchor, constant: 64.0)
```

Comment:
Why is `64` set here if it gets set by `func configure(headline: String, image: UIImage, topInset: CGFloat)`?

Reply:
good point, I removed this old default value